Dz7/Zd34.py

```diff
@@ -0,0 +1,29 @@
+# Задача 34: Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку
+# разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам
+# стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число
+# гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного
+# слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг
+# от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. В ответе
+# напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не
+# в порядке
+
+# Ввод:
+# пара-ра-рам рам-пам-папам па-ра-па-дам
+
+# Вывод:
+# Парам пам-пам
+
+vowel_letters = ("а", "о", "у", "ы", "э", "е", "ё", "и", "ю", "я")
+text = "пара-ра-рам рам-пам-папам па-ра-па-да".lower().split()
+print(f'Vowel letters: {vowel_letters}')
+
+
+def ritm(text, vowel_letters):
+    def count(x): return sum(1 for i in x if i in vowel_letters)
```

Comment:
Почему вы объявили эту функцию внутри функции?

Reply:
Такая запись, хоть и допускается, но не является традиционной. Кроме того, можно сделать чуть проще
```suggestion
    def count(x):
        return sum(i in vowel_letters for i in x)
```